conda.yml

```diff
@@ -1,4 +1,4 @@
-name: rxn_aa_mapper
+name: rxn_aa_mapper2
```

Comment:
?? restore normal name

Reply:
I set it back to the original, I did this to creare a copy of the repo and work and that copy tracking the modifications I have done before. The Conda file is not changed